src/Tribe/Ticket_Object.php

```diff
@@ -1021,6 +1030,39 @@ public function get_event() {
 			return null;
 		}
 
+		/**
+		 * Returns whether the ticket type should show on
+		 * the front page and in emails. Defaults to true.
+		 *
+		 * @since TBD
+		 *
+		 * @return boolean
+		 */
+		public function show_type() {
+			/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
+			$tickets_handler = tribe( 'tickets.handler' );
+
+			$key = $tickets_handler->key_show_type;
+
+			$show = true;
+			if ( metadata_exists( 'post', $this->ID, $key ) ) {
+				$show = get_post_meta( $this->ID, $key, true );
+			}
+
+			/**
+			 * Allows filtering of the value so we can for example, disable it for a theme/site
+			 *
+			 * @since TBD
+			 *
+			 * @param boolean whether to show the type or not.
+			 * @param int ticket ID.
+			 */
+			$show = apply_filters( 'tribe_tickets_show_type', $show, $this->ID );
```

Comment:
```suggestion
			/**
			 * Allows filtering of the value so we can for example, disable it for a theme/site
			 *
			 * @since TBD
			 *
			 * @param boolean                       $show   Whether to show the type or not.
			 * @param Tribe__Tickets__Ticket_Object $ticket The ticket object.
			 */
			$show = apply_filters( 'tribe_tickets_show_type', $show, $this );
```